CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## dev
 
+## 5/14/24
+* Adds guid and created fields to PhenotypePrioritization model (REQUIRES DB MIGRATION)
```

Comment:
PRs to dev should have new changelog entries go in the dev section, not have a date. When seqr is released from dev to master we add a date to the changelog to indicate the dte all the rleevent features were actually released

Reply:
ah ok I just fixed it